plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/aspects/StashWindowHandles.java

```diff
@@ -0,0 +1,23 @@
+package ru.sbtqa.tag.pagefactory.web.aspects;
+
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.annotation.Pointcut;
+import org.openqa.selenium.WebDriver;
+import ru.sbtqa.tag.datajack.Stash;
+import ru.sbtqa.tag.pagefactory.context.PageContext;
+
+@Aspect
+public class StashWindowHandles {
+
+    @Pointcut("call(* org.openqa.selenium.WebElement.click())")
+    public void click() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)